tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+extern gpio_t DASHBOARD_POWER;
+extern gpio_t SERVICE_SECTION_POWER;
+extern gpio_t THROTTLE_POWER;
+
+void init_devices();
```

Comment:
also, in a real api.h file, I would be adding in docstrings so someone who wants to use this library can see how